test/json-server-data/templates.json

```diff
@@ -57,9 +57,9 @@
dragid=\"jsPlumb_1_5\" elid=\"jsPlumb_1_1\"><svg style=\"position:absolute;left:0px;top:0px\" width=\"20\" height=\"20\" pointer-events=\"all\" position=\"absolute\" version=\"1.1\"\n      xmlns=\"http://www.w3.org/1999/xhtml\"><circle cx=\"10\" cy=\"10\" r=\"10\" version=\"1.1\"\n      xmlns=\"http://www.w3.org/1999/xhtml\" fill=\"#666\" stroke=\"none\" style=\"\"></circle></svg></div><div class=\"task ui-draggable _jsPlumb_endpoint_anchor_\" id=\"jsPlumb_1_8\" style=\"top: 200px; left: 200px;\"><a class=\"task-name\"><img src=\"http://localhost:9000/studio/images/Groovy.png\" width=\"20px\">&nbsp;<span class=\"name\">Task2</span></a></div><div class=\"_jsPlumb_endpoint source-endpoint dependency-source-endpoint connected _jsPlumb_endpoint_anchor_ ui-draggable ui-droppable\" style=\"position: absolute; height: 20px; width: 20px; left: 240px; top: 230px;\"><svg style=\"position:absolute;left:0px;top:0px\" width=\"20\" height=\"20\" pointer-events=\"all\" position=\"absolute\" version=\"1.1\"\n      xmlns=\"http://www.w3.org/1999/xhtml\"><circle cx=\"10\" cy=\"10\" r=\"10\" version=\"1.1\"\n      xmlns=\"http://www.w3.org/1999/xhtml\" fill=\"#666\" stroke=\"none\" style=\"\"></circle></svg></div><div class=\"_jsPlumb_endpoint target-endpoint dependency-target-endpoint _jsPlumb_endpoint_anchor_ ui-draggable ui-droppable _jsPlumb_endpoint_connected connected\" style=\"position: absolute; height: 20px; width: 20px; left: 240px; top: 190px;\"><svg style=\"position:absolute;left:0px;top:0px\" width=\"20\" height=\"20\" pointer-events=\"all\" position=\"absolute\" version=\"1.1\"\n      xmlns=\"http://www.w3.org/1999/xhtml\"><circle cx=\"10\" cy=\"10\" r=\"10\" version=\"1.1\"\n      xmlns=\"http://www.w3.org/1999/xhtml\" fill=\"#666\" stroke=\"none\" style=\"\"></circle></svg></div><svg style=\"position:absolute;left:145.5px;top:138.5px\" width=\"115\" height=\"62\" pointer-events=\"none\" position=\"absolute\" version=\"1.1\"\n      xmlns=\"http://www.w3.org/1999/xhtml\" class=\"_jsPlumb_connector \"><path d=\"M 94 61 C 104 11 -10 50 0 0 \" transform=\"translate(10.5,0.5)\" pointer-events=\"visibleStroke\" version=\"1.1\"\n      xmlns=\"http://www.w3.org/1999/xhtml\" fill=\"none\" stroke=\"#666\" style=\"\"></path><path pointer-events=\"all\" version=\"1.1\"\n      xmlns=\"http://www.w3.org/1999/xhtml\" d=\"M85.719017,39.32304425 L70.12705316271143,24.974153571739002 L72.86391419173634,33.77810675810958 L64.582115670821,37.829256380002654 L85.719017,39.32304425\" class=\"\" stroke=\"#666\" fill=\"#666\" transform=\"translate(10.5,0.5)\"></path><path pointer-events=\"all\" version=\"1.1\"\n      xmlns=\"http://www.w3.org/1999/xhtml\" d=\"M85.719017,39.32304425 L70.12705316271143,24.974153571739002 L72.86391419173634,33.77810675810958 L64.582115670821,37.829256380002654 L85.719017,39.32304425\" class=\"\" stroke=\"#666\" fill=\"#666\" transform=\"translate(10.5,0.5)\"></path></svg></div></body></html>\n ]]>\n    </visualization>\n  </metadata>\n</job>",
```

Comment:
I am a bit concerned about the urls generated by the visualization, e.g.:
http://localhost:9000/studio/images/Javascript.png
Would this not create portability issues of the workflow ?

Reply:
@fviale yes, it would (heppened to me), Chloe is fixing this